components/public-api-server/pkg/apiv1/identityprovider_test.go

```diff
@@ -55,6 +61,11 @@ func TestGetIDToken(t *testing.T) {
 				ma.EXPECT().GetLoggedInUser(gomock.Any()).Return(
 					&protocol.User{
 						Name: "foobar",
+						Identities: []*protocol.Identity{
+							nil,
+							{Deleted: true, PrimaryEmail: "nonsense@gitpod.io"},
+							{PrimaryEmail: "correct@gitpod.io"},
```

Comment:
```suggestion
							{Deleted: false, PrimaryEmail: "correct@gitpod.io"},
```

I know it's the default, but visually grepping the tests the symmetry makes it easier